pkg/cmd/flags/output.go

```diff
@@ -13,16 +14,19 @@ import (
 
 func outputHelp() string {
 	return `Control how and where output is printed.
-Possible options:
-[format:]table                                     output events in table format
-[format:]table-verbose                             output events in table format with extra fields per event
-[format:]json                                      output events in json format
-[format:]gob                                       output events in gob format
-[format:]gotemplate=/path/to/template              output events formatted using a given gotemplate file
-forward:url                                        send events in json format using the Forward protocol to a Fluent receiver
-out-file:/path/to/file                             write the output to a specified file. create/trim the file if exists (default: stdout)
-log-file:/path/to/file                             write the logs to a specified file. create/trim the file if exists (default: stderr)
+Format options:
+table[:/path/to/file,...]                          output events in table format (default). The default path to file is stdout.
+table-verbose[:/path/to/file,...]                  output events in table format with extra fields per event. The default path to file is stdout.
+json[:/path/to/file,...]                           output events in json format. The default path to file is stdout.
+gob[:/path/to/file,...]                            output events in gob format. The default path to file is stdout.
+gotemplate=/path/to/template[:/path/to/file,...]   output events formatted using a given gotemplate file. The default path to file is stdout.
 none                                               ignore stream of events output, usually used with --capture
+
+Log options:
+log-file:/path/to/file                             write the logs to a specified file. create/trim the file if exists (default: stderr)
+forward:url                                        send events in json format using the Forward protocol to a Fluent receiver
```

Comment:
is this a log option (since it is located here) or an output printer?

Reply:
@yanivagman output printer.